apps/api-portal/nginx.conf

```diff
@@ -0,0 +1,48 @@
+events {
```

Comment:
what makes this nginx config different to avoid using the [shared](https://github.com/GlobalFishingWatch/frontend/blob/develop/config/nginx.conf#L2) one?

Reply:
Every app docker image build process is using their own `nginx.conf` (eg: [fishing map](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/project.json#L19)) that is not aware of other apps paths and they are located in the app's root folder (eg:  [fishing-map nginx.conf](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/nginx.conf)). The [shared nginx.conf ](https://github.com/GlobalFishingWatch/frontend/blob/develop/config/nginx.conf#L2) is used only in the root [`docker-compose.yaml`](https://github.com/GlobalFishingWatch/frontend/blob/develop/docker-compose.yaml#L75) to configure our local environments the most similar to prod, having the apps running in specific paths and not the root.